### ISSUE TITLE
Making Project Sidewalk website accessible to screen readers?

### ISSUE THREAD

**Issue by jonfroehlich** [ProjectSidewalk/SidewalkWebpage#2904]
We have received multiple requests to make the Project Sidewalk website accessible to people who are blind and low vision (from Joy Hammel, from the National Mobility Center, from Libe Aceso). While I completely agree in theory and philosophically, this is a complex ask not least because Project Sidewalk is, fundamentally, about using visual search to find and identify accessibility problems. The task requires vision.

Still, we could ensure that other parts of the website are accessible (to the degree we can).

Libre Aceso said:

> También preguntó si el sitio web podría modificarse para que sea accesible a personas ciegas. Ellos (libre acceso) no dan ese servicio, pero podrían pasarnos el contacto de un consultor que puede dar este servicio

Translation:

> He also asked if the website could be modified to be accessible to blind people. They (free access) do not provide this service, but they could pass us the contact of a consultant who can provide this service

**Comment by misaugstad** [comment]
Related to #2522

**Comment by misaugstad** [comment]
Jon mentioned talking to someone who has a hard time using a keyboard and their experience using Sidewalk. They suggested:

>checking compatibility with Apple speech so that he could at least say "Mark 3, 'no alternate route'"

**Comment by jonfroehlich** [comment]
They also suggested having a button that let's them do 90 degree turns as panning was difficult.
